### DISCUSSION TITLE
Boot Jetson Orin Nano from SD

### DISCUSSION THREAD

**Discussion by pedroagan** [OE4T/meta-tegra#1766] [Q&A]
Hello,
I have generated a custom jetson nano orin image based on sato and the image does not boot, in the boot menu of jetson I view the following error:
![image](https://github.com/user-attachments/assets/be4454c5-033e-44ec-9efe-d05b1c42c94b)

The branch that I am using is Scarthgap but with Scarthgap-l4t-r35x I have started with the jetson and same configuration without problems.

Is it necessary change something in the local.conf to solve this problem?

Thanks in advance. Regards,

**Comment by madisongh** [comment]
From the UEFI version displayed in the picture, it looks like you skipped flashing the boot firmware.

**Reply by pedroagan** [reply]
Thank you for your answer. How can flash the boot firmware? Sorry for the dummy question, I am new with jetson...

**Reply by dwalkes** [reply]
@pedroagan see https://github.com/OE4T/meta-tegra/wiki/Flashing-the-Jetson-Dev-Kit
